jitbuilder/apigen/cppgen.py

```diff
@@ -773,7 +776,11 @@ def write_common_decl(self, writer, api_desc):
         writer.write("#define {}_INCL\n\n".format(api_desc.project()))
 
         # write some needed includes and defines
-        writer.write("#include <stdint.h>\n\n")
+
```

Comment:
To (try to) keep the style consistent, please remove this blank line between the comment and the code.

Reply:
Fixed.